docs/components/concepts/clusters.md

```diff
@@ -32,19 +32,27 @@ When your Free Trial plan expires, you are automatically transferred to the Free
 
 ## Development clusters
 
-Development clusters are recommended for development, testing, proof of concepts, and demos.
+Development clusters, available in the Starter and Enterprise plans, are recommended for development, testing, proof of concepts, and demos.
 
-Starter Plan users have one **development cluster**, with free execution for development included in their plan. Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+The way this type of cluster works varies depending if you are using it in the Starter or the Enterprise plan.
```

Comment:
@christian-konrad Do we want to capitalize "Plan" for "Enterprise Plan", for example, or leave as "Enterprise plan"? I don't have strong opinions here, but can make a styling decision if necessary.

Reply:
Did you mean to tag @jprcoliveira?

Reply:
@christinaausley feel free to make a call here to ensure we follow the correct styling here and are consistent :)